sys/arm64/arm64/efirt_support.S

```diff
@@ -70,11 +70,15 @@ ENTRY(efi_rt_arch_call_nofault)
 	ldr	c6, [c0, #EC_FPTR]
 
 	/* Jump to 0f + 4 * (5 - ec_argcnt) + 1 */
+#ifdef __ARM_MORELLO_PURECAP_BENCHMARK_ABI
+	adr	c9, 0f
+#else
 	adr	c9, 0f + 1
+#endif
```

Comment:
```suggestion
	adr	c9, 0f
```

I would also make this change along with the br below as its own commit since it also applies to purecap, and aligns the code with what Morello LLVM generates these days